### PR TITLE
Replace XDG related functions

### PR DESCRIPTION
**Pull request by izidormatusov**
Fix for #392

### CODE REVIEW

File: GTG/backends/unmaintained/backend_rtm.py [hunk 1]
```diff
@@ -562,7 +562,7 @@ def _authenticate(self):
             if not self.token:
                 self.rtm = createRTM(
                     self.PUBLIC_KEY, self.PRIVATE_KEY, self.token)
-                subprocess.Popen(['xdg-open', self.rtm.getAuthURL()])
+                webbrowser.open(self.rtm.getAuthURL())
```
Comment (leio) [inline GTG/backends/unmaintained/backend_rtm.py:565]: Use Gtk.show_uri instead?

This is just a comment on this bit, not a full review

Comment (diegogangl) [reply GTG/backends/unmaintained/backend_rtm.py:565]: This should be `Gio.AppInfo.launch_default_for_uri()` ([link](https://lazka.github.io/pgi-docs/#Gio-2.0/classes/AppInfo.html#Gio.AppInfo.launch_default_for_uri)) so it uses the right portal in the flatpak version.

Comment (izidormatusov) [reply GTG/backends/unmaintained/backend_rtm.py]: Used `Gio.AppInfo.launch_default_for_uri()`

**Comment by diegogangl** [comment]
First commit of 0.5! Thanks @izidormatusov :D

**Pull request merged**